requirements/development.txt

```diff
@@ -6,12 +6,10 @@
 #    pip-compile-multi
 #
 -r base.txt
--e file:.
+-e file:///mnt/c/temp/superset_numexp
```

Comment:
remove this and replace it with `.` like it was before

Reply:
fixed it back to .